dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1024,25 +1023,25 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 			time_t expire = 1; // Default to indefinate mute
 
 			if (args.size() >= 2) {
-				uint32_t days = 0;
-				uint32_t hours = 0;
-				if (!GeneralUtils::TryParse(args[1], days)) {
+				const auto days = GeneralUtils::TryParse<uint32_t>(args.at(1));
+				const auto hours = GeneralUtils::TryParse<uint32_t>(args.at(2));
```

Comment:
This logic was moved from its previous wrapper to verify there were enough elements in the vector and will crash if this is tried to be used

Reply:
It's not though?

Reply:
Ah, the hours could potentially be.